common/src/stack/ansible/plugins/inventory/stacki.py

```diff
@@ -0,0 +1,130 @@
+# @copyright@
+# Copyright (c) 2006 - 2020 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
```

Comment:
Consider documenting what this is doing.

Reply:
No. Ansible documented what it's doing: https://docs.ansible.com/ansible/latest/dev_guide/developing_python_3.html#tips-tricks-and-idioms-for-python-2-python-3-compatibility

If you are that curious what the boilerplate is for, I'll let them explain it.